src/overlays/actors/ovl_Dm_An/z_dm_an.c

```diff
@@ -33,54 +33,65 @@ ActorInit Dm_An_InitVars = {
     (ActorFunc)NULL,
 };
 
-typedef enum {
-    /*  -1 */ DMAN_ANIM_NONE = -1,
-    /* 0x0 */ DMAN_ANIM_0,
-    /* 0x1 */ DMAN_ANIM_1,
-    /* 0x2 */ DMAN_ANIM_2,
-    /* 0x3 */ DMAN_ANIM_3,
-    /* 0x4 */ DMAN_ANIM_4,
-    /* 0x5 */ DMAN_ANIM_5,
-    /* 0x6 */ DMAN_ANIM_6,
-    /* 0x7 */ DMAN_ANIM_7,
-    /* 0x8 */ DMAN_ANIM_8,
-    /* 0x9 */ DMAN_ANIM_9,
-    /* 0xA */ DMAN_ANIM_10,
-    /* 0xB */ DMAN_ANIM_11,
-    /* 0xC */ DMAN_ANIM_12,
-    /* 0xD */ DMAN_ANIM_13,
-    /* 0xE */ DMAN_ANIM_MAX
+/**
+ * Anju stores her animations across different objects and the ones used by this actor are put together right next to
+ * each other in the sAnimationInfo array. Due to this, animation functions check which object to load by comparing
+ * index ranges. To make this a bit easier to read, this enum includes `DMAN_ANIMOBJ_*` values that mark when a range of
+ * animations of a certain object start
+ */
+typedef enum DmAnAnimation {
+    /* -1 */ DMAN_ANIM_NONE = -1,
+    /*  0 */ DMAN_ANIM_SIT_IN_DISBELIEVE,
```

Comment:
Er, the passive is "disbelief", right? 

This feels like an attempt to avoid "ing" but the current tense would be "sit**ing** in disbelief" here, I think?

Reply:
This is a combination of me messing up English verbal times and trying to avoid "ing".

Are we ok with naming this animation "sitting in disbelief"?